.gitignore

```diff
@@ -36,6 +36,7 @@ local.properties
 node_modules/
 npm-debug.log
 yarn-error.log
+package-lock.json
```

Comment:
This is not the recommended way to work with package-lock.json. According to the official NPM documentation, we are supposed to commit this file always. However, since you use yarn and I use NPM, I didn't want to mess up with any of your stuff. However, I could as well revert this change. Up to you.

Reply:
This is fine. We definitely don't want both yarn.lock and package-lock.json in the repo, since that will cause unexpected version differences.